cpp/example_code/cross-service/serverless-aurora/SES3EmailHandler.cpp

```diff
@@ -0,0 +1,212 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+/**
+ *  SES3EmailHandler.h/.cpp
+ *
+ *  The code in these 2 file implements the sending of a multipart email message containing
+ *  plain text, HTML text, and an attachment file using Amazon Simple Email Service
+ *  (Amazon SES).
+ *
+ * To run the example, refer to the instructions in the ReadMe.
+ *
+ */
+
+#include "SES3EmailHandler.h"
```

Comment:
Just curious why the '3' in the name?

Reply:
sesv2 was intended.  I will change that,